client/components/language-picker/site-language-picker.jsx

```diff
@@ -35,7 +35,7 @@ const SiteLanguagePicker = ( { languages: origLanguages, ...restProps } ) => {
 		error,
 		isLoading,
 	} = useQuery(
-		'wporg-translations-' + wpVersion,
+		[ 'wporg-translations-' + wpVersion ],
```

Comment:
Noting that now when we always use an array for query keys, we could have the library actually take care of [hashing the key](https://tanstack.com/query/v4/docs/react/guides/query-keys#query-keys-are-hashed-deterministically). Meaning, we could alter this to:

```suggestion
		[ 'wporg-translations', wpVersion ],
```

That applies to all query keys that we're currently manually hashing.

Reply:
Agreed 👍🏻 I'll do a commit which takes care of these.

Reply:
That totally makes sense to be another PR, FWIW.

Reply:
Added here https://github.com/Automattic/wp-calypso/pull/74720/commits/01d47ac5b2b5d68cca4ae4f05437774fd94779b4